packages/zowe-explorer/src/shared/init.ts

```diff
@@ -218,14 +218,22 @@ export function watchConfigProfile(context: vscode.ExtensionContext, providers:
 
     context.subscriptions.push(...watchers);
 
+    const extenderCallback = ZoweExplorerApiRegister.getInstance().getProfileChangeCallback();
+
     watchers.forEach((watcher) => {
         watcher.onDidCreate(async () => {
             ZoweLogger.info(localize("watchConfigProfile.create", "Team config file created, refreshing Zowe Explorer."));
             await vscode.commands.executeCommand("zowe.extRefresh");
+            if (extenderCallback) {
+                await extenderCallback.call(this, globals.EVENT_TYPES.CREATE);
```

Comment:
I think if the type signature was updated to match the function in `ZoweExplorerApiRegister.ts`, we can just call the callback directly w/o `this`:

`await extenderCallback(EVENT_TYPES.CREATE)`